src/translations/en.json

```
@@ -2837,6 +2837,7 @@
             "area": "In {area}",
             "no_area": "No Area",
             "not_loaded": "Not loaded",
+            "setup_in_progress": "Setup in progress",
```

Can we make this "Initializing". Setup can be confusing.
```suggestion
            "setup_in_progress": "Initializing",
```